packages/office-ui-fabric-react/src/components/__snapshots__/Nav.Basic.Example.tsx.shot

```diff
@@ -331,6 +332,7 @@ exports[`Component Examples renders Nav.Basic.Example.tsx correctly 1`] = `
                   name="Activity"
                 >
                   <a
+                    aria-label=""
```

Comment:
Will an empty string for `aria-label` be processed correctly by AT?

Reply:
should not have '' aria-label' fixed